soc/tenstorrent/tt_blackhole/tt_blackhole_smc.ld

```diff
@@ -10,6 +10,6 @@
  * Note that linker scripts do not allow expressions in this context.
  */
 #define SRAM_START 0x10000000
-#define SRAM_SIZE 0x30000
+#define SRAM_SIZE 0x70000
```

Comment:
This is a bump to 448KB, not 256KB right?

Reply:
Oh shoot - let me fix that 😅